ui/app/components/license-info.js

```diff
@@ -0,0 +1,32 @@
+import Component from '@ember/component';
+import { allFeatures } from 'vault/helpers/all-features';
+import { computed } from '@ember/object';
+
+export default Component.extend({
+  expirationTime: '',
+  startTime: '',
+  licenseId: '',
+  features: null,
+  text: '',
+  showForm: false,
+  isTemporary: computed('licenseId', function() {
+    return this.get('licenseId') === 'temporary';
+  }),
+  featuresInfo: computed('features', function() {
+    let info = [];
+    allFeatures().forEach(feature => {
+      let active = this.get('features').includes(feature) ? true : false;
+      info.push({ name: feature, active: active });
+    });
+    return info;
+  }),
+  saveModel() {},
+  actions: {
+    createModel(text) {
```

Comment:
createModel / saveModel is a bit confusing here just because we're never calling `createRecord` on the store - we'll always have a model. You should be able to have a `saveModel` in the actions hash and it won't conflict with the passed in saveModel function (though... maybe that's confusing too? 🤔) - naming things...